test/helpers.bash

```diff
@@ -260,7 +260,7 @@ function setup_crio() {
 
     RUNTIME_ROOT=${RUNTIME_ROOT:-"$TESTDIR/crio-runtime-root"}
     # export here so direct calls to crio later inherit the variable
-    export CONTAINER_RUNTIMES=${CONTAINER_RUNTIMES:-$CONTAINER_DEFAULT_RUNTIME:$RUNTIME_BINARY_PATH:$RUNTIME_ROOT:$RUNTIME_TYPE}
+    export CONTAINER_RUNTIMES=${CONTAINER_RUNTIMES:-$CONTAINER_DEFAULT_RUNTIME:$RUNTIME_BINARY_PATH:$RUNTIME_ROOT:$RUNTIME_TYPE:$PRIVILEGED_WITHOUT_HOST_DEVICES}
```

Comment:
Oooi!

Here you're using `$PRIVILEGED_WITHOUT_HOST_DEVICES`, but you didn't define it.
Take a look all the way up in the file and check how, for instance, RUNTIME_TYPE is defined.

Reply:
Thanks - just fixed it